### PR TITLE
solution

### PR DESCRIPTION
**Pull request by pasha1932**
[DEMO LINK](https://pasha1932.github.io/react_static-list-of-posts-js/)

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,104 +1,27 @@
...
+  return usersFromServer.find(user => user.id === userId) || null;
+}
 
-            <a className="UserInfo" href="mailto:Julianne.OConner@kory.org">
-              Patricia Lebsack
-            </a>
-          </p>
-        </div>
+function getCommentsById(id) {
+  return commentsFromServer.filter(comment => comment.postId === id) || null;
```
Comment (etojeDenys) [inline src/App.jsx:13]: null is redundant here, 'cause the filter method always returns an array

File: src/components/PostInfo/PostInfo.jsx [hunk 2]
```diff
@@ -1 +1,53 @@
...
+      <p className="PostInfo__body">
+        est rerum tempore vitae sequi sint nihil reprehenderit dolor beatae ea
+        dolores neque fugiat blanditiis voluptate porro vel nihil molestiae ut
+        reiciendis qui aperiam non debitis possimus qui neque nisi nulla
+      </p>
+
+      <hr />
+
+      <b data-cy="NoCommentsMessage">No comments yet</b>
+    </div> */}
```
Comment (etojeDenys) [inline src/components/PostInfo/PostInfo.jsx:29]: remove all comments

File: src/components/PostList/PostList.jsx [hunk 3]
```diff
@@ -1 +1,11 @@
-export const PostList = () => <>Put the list here</>;
+import { PostInfo } from '../PostInfo/PostInfo';
+
+export const PostList = ({ posts }) => (
+  <>
```
Comment (etojeDenys) [inline src/components/PostList/PostList.jsx:4]: redundant fragment 

https://react.dev/reference/react/Fragment#usage

File: src/components/PostList/PostList.jsx [hunk 4]
```diff
@@ -1 +1,11 @@
...
+    <div className="PostList">
+      {posts.map(post => (
+        <PostInfo post={post} />
```
Comment (etojeDenys) [inline src/components/PostList/PostList.jsx:7]: add the key prop here

File: src/components/UserInfo/UserInfo.jsx [hunk 5]
```diff
@@ -1 +1,7 @@
-export const UserInfo = () => <>Put the user here</>;
+export const UserInfo = ({ user }) => (
+  <>
```
Comment (etojeDenys) [inline src/components/UserInfo/UserInfo.jsx:2]: fix it everywhere

**Review by etojeDenys — CHANGES_REQUESTED** [review]

**Review by l4st1m0za — APPROVED** [review]
Good job!

**Pull request closed without merging**